src/Form.js

```diff
@@ -55,13 +57,18 @@ export default class Form extends Component {
     }
   }
   onSubmit = (event) => {
-    const { onSubmit } = this.props;
+    const { onSubmit, onValidationError } = this.props;
     const { formState, resource } = this.state;
+    let formErrors = {};
     event.preventDefault();
     if (formState.isValid()) {
       onSubmit(resource);
     } else {
-      this.setState({ isFormSubmitted: true, errors: formState.getErrors() });
+      formErrors = formState.getErrors();
+      if (typeof onValidationError === 'function') {
+        onValidationError(formErrors);
+      }
+      this.setState({ isFormSubmitted: true, errors: formErrors });
```

Comment:
I believe this can be simplified to be `this.setState({ isFormSubmitted: true, errors: formErrors });`